packages/plugins/type-designer/src/components/drawer-content.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+// CONSTANTS
+import { SCD_ELEMENTS } from '@oscd-plugins/core'
+// SMUI COMPONENTS
+import Textfield from '@smui/textfield'
+import Button from '@smui/button'
+// STORES
+import { pluginStore } from '@/stores/plugin.store'
+import { drawerStore } from '@oscd-plugins/ui'
+// TYPES
+import type { DataTypeTemplates } from '@oscd-plugins/core'
+
+//====== INITIALIZATION ====//
+
+// props
+export let typeElement: DataTypeTemplates.TypeElement
+export let currentColumn: keyof typeof SCD_ELEMENTS
+
+let localAttributes: Record<(typeof standardAttributes)[number], string>
```

Comment:
Same issue here, this typing looks very fragile.

Reply:
See last comment